src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final int index = new Random().nextInt(Color.values().length);
```

Comment:
You should have field `index` inside `getRandomColor()` method.
But also make `new Random()` as class field